core/templates/pages/signup-page/signup-page.component.ts

```diff
@@ -18,15 +18,19 @@
 
 require('base-components/base-content.directive.ts');
 require(
-  'components/common-layout-directives/common-elements/' +
-  'confirm-or-cancel-modal.controller.ts');
+  'pages/signup-page/modal-templates/license-explanation-modal.controller.ts');
+require(
+  'pages/signup-page/modal-templates/' +
+  'registration-session-expired-modal.controller.ts');
+
 require('domain/utilities/url-interpolation.service.ts');
 require('services/alerts.service.ts');
 require('services/id-generation.service.ts');
 require('services/site-analytics.service.ts');
 require('services/user.service.ts');
 require('services/contextual/url.service.ts');
 require('services/stateful/focus-manager.service.ts');
+require('constants.ts');
```

Comment:
Why is this added?

Reply:
I think this `require` is not necessary, I'm removing it.